fabfile.py

```diff
@@ -1662,6 +1662,20 @@ def docker_compose():
     # run("docker-compose -f docker/build/docker-compose.yml up")
 
 
+@task
+def get_ssl_certificates():
+    if env.oscp_path:
+        root_certificate = run('curl https://letsencrypt.org/certs/isrgrootx1.pem.txt')
+        intermediate_certificate_1 = run('curl https://letsencrypt.org/certs/lets-encrypt-x3-cross-signed.pem.txt')
+        intermediate_certificate_2 = run('curl https://letsencrypt.org/certs/letsencryptauthorityx3.pem.txt')
+        with open(env.oscp_path, 'w') as certificates_file:
+            for certificate_file in [root_certificate, intermediate_certificate_1, intermediate_certificate_2]:
+                certificates_file.write(certificate_file)
+                certificates_file.write('\n')
+    else:
+        print("Can't get ssl certificates, env.oscp_path does not exist")
```

Comment:
Use `print(yellow("....")` And it's not that it does not exist, the `oscp_path` is not set.